javascript-methods/main.js

```diff
@@ -0,0 +1,68 @@
+let x = 5;
+let y = 8;
+let z = 12;
+let maximumValue = Math.max(x, y, z);
+console.log(maximumValue);
+
+const heroes = [
+  'Batman',
+  'Superman',
+  'Spiderman',
+  'Ironman',
+];
+
+let randomNumber = Math.random();
+randomNumber = randomNumber * heroes.length;
+
+let randomIndex = Math.floor(randomNumber);
```

Comment:
This should be `const`

Reply:
Getting an error when I use const for randomNumber.

Reply:
This comment was referring to `randomIndex` not `randomNumber`. Sorry for any confusion!